index.html

```diff
@@ -30,10 +47,26 @@
           }
           const r = res.get(key);
           r.data.push(value);
-          r.revision.push(entry.revision.substr(0, 6));
+          r.revision.push(entry.revision.substr(0, 7));
         }
       }
-      return res
+      return res;
+    }
+
+    function show_clipboard_text(text) {
+      var notifyClipboardTextElem = document.getElementById("notify-clipboard-text");
+      notifyClipboardTextElem.textContent = text + " copied to clipboard";
+      notifyClipboardTextElem.className = 'inline-text';
+      setTimeout(function () {
+        notifyClipboardTextElem.className = 'hidden';
```

Comment:
```suggestion
        notifyClipboardTextElem.classList.add("hidden");
```

Reply:
Changed to only add and remove hidden class as needed